draft-ietf-cats-usecases-requirements.md

```diff
@@ -888,9 +888,32 @@ elements in the participating edges.
 
 R7: MUST include network metrics.
 
+To better understand the meaning of different metrics and to better	
+support appropriate use of metrics,	
 
-## Support Moderate Metric Distributing
+R8: a computing semantic model SHOULD be defined for the mapping	
```

Comment:
Delete it.

Reply:
A second thought after our discussion. I think R8 is a good requirement because the notion of having a model is relevant. But i think the above needs a bit of rewording. I would say:

"R8. A model of the compute and network resources MUST be defined. Such a model MUST characterize how metrics are abstracted out from the compute and network resources. We refer to this model as the Resource Model."

Then I would also add:

"R9: The Resource Model MUST be implementable in an interoperable manner. That is, independent implementations of the Resource Model must be interoperable."

"R9: The Resource Model MUST be executable in a scalable manner. That is, an agent implementing the Resource Model must be able to execute it at the required time scale and at an affordable cost (e.g., memory footprint, energy, etc.)"

"R10: The Resource Model MUST be useful. That is, the metrics that an agent can obtain by executing the Resource Model must be useful to make node and path selection decisions."